danielMunteanu/src/main/java/danielCollectionTask/CompanyOrder.java

```diff
@@ -0,0 +1,55 @@
+package danielCollectionTask;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static java.time.LocalDateTime.now;
+
+public class CompanyOrder implements Order {
```

Comment:
This method should extend the AbstractOrder, therefore it will explicitly implement the Order interface